quarterback-packages/api/src/common/jwt.svc.ts

```diff
@@ -18,7 +18,7 @@ const { sign, verify } = pkg
 
 import { Maybe, User } from '@manuscripts/quarterback-types'
 
-import { IJwtPayload } from '$typings/auth'
+import { IJwtPayload } from '../typings/auth'
```

Comment:
@mbetamony this is related to making @manuscripts/transform work in the api. I changes ts config and didn't want to preserver those aliases. It's not particularly useful and only confusing